docs/04.INTERNALS.md

```diff
@@ -135,57 +135,61 @@ Byte-codes can be placed into four main categories.
 
 Byte-codes of this category serve for placing objects onto the stack. As there are many instructions representing multiple atomic tasks in CBC, there are also many instructions for pushing objects onto the stack according to the number and the type of the arguments. The following table list a few of these opcodes with a brief description.
 
-<div class="CSSTableGenerator" markdown="block">
+<span class="CSSTableGenerator" markdown="block">
 
-| byte-code                            | description                                       |
+| byte-code                            | description |
+| ------------------------------------ | ----------- |
```

Comment:
Sorry, but I don't fully understand, how should I align the pipes? To the bigger column in the table or to the average line width? Now they are aligned to the average line width.